SU2_CFD/src/solver_direct_transition.cpp

```diff
@@ -335,8 +335,9 @@ void CTransLMSolver::Viscous_Residual(CGeometry *geometry, CSolver **solver_cont
     numerics->SetTransVar(nodes->GetSolution(iPoint), nodes->GetSolution(jPoint));
     numerics->SetTransVarGradient(nodes->GetGradient(iPoint), nodes->GetGradient(jPoint));
     
-    numerics->SetConsVarGradient(solver_container[FLOW_SOL]->GetNodes()->GetGradient(iPoint),
-                                 solver_container[FLOW_SOL]->GetNodes()->GetGradient(jPoint));
+//    ToDo: The flow solvers do not use the conservative variable gradients
```

Comment:
here

Reply:
This one is probably a good question for @vdweide, since he has been updating the transition model in a separate branch. It looks like the viscous numerics is mostly commented out in the develop branch. I expect we should modify this to use the primitive gradients.

Reply:
No need to worry about the transition model at the moment. Since the data structures have changed so much lately, it is pretty much impossible to carry out a merge between the transition model branch and the develop branch. The implementation must be redone. @economon, we agreed that we would wait with that until PR 777 is merged in. I assume this is still the case.

Reply:
Sounds good - thanks for confirming @vdweide. Agreed we'll work on this after #777